proto/launch/params.proto

```diff
@@ -9,7 +9,11 @@ option go_package = "github.com/tendermint/spn/x/launch/types";
 message Params {
   option (gogoproto.goproto_stringer) = false;
 
-  uint64 minLaunchTime = 1 [(gogoproto.moretags) = "yaml:\"min_launch_time\""];
-  uint64 maxLaunchTime = 2 [(gogoproto.moretags) = "yaml:\"max_launch_time\""];
-  int64 revertDelay = 3   [(gogoproto.moretags) = "yaml:\"revert_delay\""];
-}
\ No newline at end of file
+  LaunchTimeRange launchTimeRange = 1 [(gogoproto.nullable) = false];
+  int64 revertDelay = 2;
+}
+
+message LaunchTimeRange {
+  uint64 minLaunchTime = 1;
+  uint64 maxLaunchTime = 2;
```

Comment:
What do you think about setting the type for both fields to `int64` since they represent Unix timestamp?
We should then add check for `minLaunchTime>0`

Then we can use `int64` for `remainingTime` in `MsgTriggerLaunch` and prevent from doing the `int64` conversion that can bring security problematics

Reply:
Do we want also to allow that `MaxLaunchTime == MinLaunchTime`?